styles.css

```diff
@@ -60,6 +71,13 @@ footer p {
 
 .page-title {
   font-size: 2rem;
+  font-variation-settings: "wght" 900;
+  text-align: center;
+  margin: 4rem 0 1.8rem 0;
+}
+
+.description {
+  color: #7a7a7a;
```

Comment:
we should use a color variable here to stay consistent with the rest of the file